.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -438,6 +440,16 @@ stages:
         artifact: linux-profiler-home-$(baseImage)
         path: $(monitoringHome)
 
+    - task: DownloadPipelineArtifact@2
+      displayName: Download dd-dotnet $(artifactSuffix)
+      inputs:
+        artifact: dd-dotnet-$(artifactSuffix)
+        path: $(monitoringHome)/$(artifactSuffix)
+
+    - script: |
+        chmod +x $(monitoringHome)/$(artifactSuffix)/dd-dotnet
```

Comment:
FWIW, we could do this in Nuke if you want to move more logic there, as part of the `ZipMonitoringHome` step... tbh, that seems like a good idea. + we could cshmod the createlogfile script if we aren't already 🤔